test/unit/search/query_components/core_query_test.rb

```diff
@@ -20,27 +20,13 @@ class CoreQueryTest < ShouldaUnitTestCase
     end
   end
 
-  context "when ab testing minimum should match" do
-    should "use the default value when no variant is passed in" do
-      builder = QueryComponents::CoreQuery.new(search_query_params)
-
-      query = builder.payload
-      assert_match(/"2<2 3<3 7<50%"/, query[:bool][:must].to_s)
-    end
-
-    should "use variant b when it is passed in" do
+  context "minimum should match" do
+    should "is part of the should clause" do
       builder = QueryComponents::CoreQuery.new(search_query_params(ab_tests: { search_match_length: 'B' }))
```

Comment:
I'm going to make these two changes and push a new version so that we can deploy this along with the new A/B test. I've checked that that's OK with @MatMoore.